crates/forge/bin/cmd/create.rs

```diff
@@ -158,10 +158,28 @@ impl CreateArgs {
             let ZkContractArtifact { bytecode, hash, factory_dependencies, abi, .. } = artifact;
 
             let abi = abi.expect("Abi not found");
-
             let bin = bytecode.expect("Bytecode not found");
+
+            let bytecode = match bin.object() {
+                BytecodeObject::Bytecode(bytes) => bytes.to_vec(),
+                _ => {
+                    let link_refs = bin
+                        .missing_libraries
+                        .iter()
+                        .map(|library| {
+                            let mut parts = library.split(':');
+                            let path = parts.next().unwrap();
+                            let name = parts.next().unwrap();
+                            format!("\t{name}: {path}")
+                        })
+                        .collect::<HashSet<String>>()
+                        .into_iter()
+                        .collect::<Vec<String>>()
+                        .join("\n");
+                    eyre::bail!("Dynamic linking not supported in `create` command - deploy the following library contracts first, then provide the address to link at compile time\n{}", link_refs)
```

Comment:
Just a chunk of good news: we're already working on it, and a test build must be ready this week!